hw/bsp/bluepill/bsp.yml

```diff
@@ -0,0 +1,69 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+### Entries containing `XXX` need to be replaced with actual values.
+
+bsp.arch: cortex_m3
+bsp.compiler: '@apache-mynewt-core/compiler/arm-none-eabi-m3'
+
+bsp.linkerscript:
+    - 'hw/bsp/bluepill/bluepill.ld'
+    - '@apache-mynewt-core/hw/mcu/stm/stm32f1xx/stm32f103.ld'
+
+bsp.linkerscript.BOOT_LOADER.OVERWRITE:
+    - 'hw/bsp/bluepill/boot-bluepill.ld'
+    - '@apache-mynewt-core/hw/mcu/stm/stm32f1xx/stm32f103.ld'
+bsp.part2linkerscript: "hw/bsp/bluepill/split-bluepill.ld"
```

Comment:
Not sure why this is here, and `split-bluepill.ld` doesn't seem to be part of this PR. Missed including it?

Reply:
not sure where I got this from, removed.